java/test/jmri/LightTest.java

```diff
@@ -31,6 +37,26 @@ public void testTransitionConstants() {
         Assert.assertTrue("TRANSITIONINGTOFULLOFF overlap", (Light.TRANSITIONINGTOFULLOFF & Light.TRANSITIONING) != 0);
     }
 
+    public void testLight() {
```

Comment:
I think this test would be better as an addition to jmri.impelmentation.AbstractLightTestBase.  That way it gets run against all of the actual light implementations and not the MyLight class you added here.

Reply:
I agree. Fixed.